src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Telemetry/ProjectTelemetryEvent.cs

```diff
@@ -45,5 +64,7 @@ public class ProjectTelemetryEvent : TelemetryEvent
         /// True, if project can be upgraded to PackageReference.
         /// </summary>
         public bool IsProjectPRUpgradable => (bool)base[IsPRUpgradable];
+
+        public string FullPath => (string)GetPiiData().Where(kv => kv.Key == nameof(FullPath)).First().Value;
```

Comment:
Wouldn't this throw when the full path is not set?

Reply:
Yes. It will be removed on next changeset.